src/tabs/UnitTest/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { Results, SuiteResult } from '../../bundles/unittest/types';
+
+/**
+ * Tab for unit tests.
+ * @author Jia Xiaodong
+ */
+
+type Props = {
+  result: any;
+};
+
+class UnitTests extends React.PureComponent<Props> {
+  /**
+   * Converts the results of a test suite run into a table format in its own div.
+   */
+  private static suiteResultToDiv(suiteResult: any) {

```

Comment:
Is it possible to restrict the type of `suiteResult` parameter here and try to move the type casting `... as SuiteResult` all the way to the point in which it is extracted from `this.props`? Reason is because we want to avoid dealing with `any` types as much as possible.